src/cmd/troubleshoot/oneagentapm.go

```diff
@@ -0,0 +1,25 @@
+package troubleshoot
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/pkg/errors"
+)
+
+func checkOneAgentAPM(ctx *troubleshootContext) error {
+	log = newTroubleshootLogger("oneAgentAPM")
+
+	logNewCheckf("checking if OneAgentAPM object exists ...")
+
+	exists, err := kubeobjects.CheckIfOneAgentAPMExists(ctx.apiReader)
+
+	if err != nil {
+		return err
+	}
+
+	if exists {
+		return errors.New("OneAgentAPM object still exists")
```

Comment:
more information with some points how to fix the problem could be helpful here

Reply:
thanks, is there another way than to delete those objects?

Reply:
oneagent operator needs to be fully uninstalled by the customer